assets/css/_map_page.scss

```diff
@@ -0,0 +1,102 @@
+.m-map-page {
+  display: flex;
+  flex: 1 1 auto;
+  flex-direction: row;
+  height: 100%;
+  position: relative;
+  overflow: visible;
+}
+
+.m-map-page__map {
+  display: flex;
+  flex: 1 1 auto;
+  position: relative;
+}
+
+.m-map-page__input-and-results {
+  background-color: $color-bg-base;
+  display: flex;
+  flex: 0 0 auto;
+  flex-direction: column;
+  width: 22rem;
+}
+
+.m-map-page__input,
+.m-search-display {
+  padding: 1.5rem 1.1875rem;
+}
+
+.m-map-page__input {
+  border-bottom: 1px solid $white;
+}
+
+.m-map-page__toggle-mobile-display-button {
+  @include button-text;
+  display: none;
+  font-size: 0.875rem;
+  margin: 0.5rem auto 0;
+}
+
+.m-search-display {
+  flex: 1 1 auto;
+  overflow-y: scroll;
+}
+
+@media screen and (max-width: $mobile-max-width) {
+  .m-map-page {
+    flex-direction: column;
+  }
+
+  .m-map-page--show-list {
+    .m-map-page__input-and-results {
+      flex: 1 1 auto;
+      height: 100%;
+    }
+
+    .m-map-page__map {
+      // Needs to be hidden instead of not displayed so the map still gets drawn properly
+      position: fixed;
+      top: 100vh;
+    }
```

Comment:
thought: I'm curious about this comment, would `visibility: hidden;` work here? 
https://developer.mozilla.org/en-US/docs/Web/CSS/visibility

I'm not really sure what this is doing though, so I guess either I'd need to see it or the comment could possibly be made more clear?

Reply:
There seems to be something else at play here - I tried changing this to hidden but the map displayed very zoomed in when using mobile dimensions. 

This (and most of the styles in _map_page) were copied from the original styles in [_search_page.scss](https://github.com/mbta/skate/blob/493b9ec20e7896c32224d5fde00c2792a161d7b4/assets/css/_search_page.scss#L56) - I tried to keep the changes minimal beyond that for the purposes of only getting the new card positioned correctly relative to the existing elements.